cross-modal-search/app.py

```diff
@@ -87,10 +88,11 @@ def main(task, num_docs, request_size, data_set, model_name):
     print(f'### task = {task}')
```

Comment:
not sure why the previous PR was merged with these prints, but can you remove them?

Reply:
I can, but it looks like they are all over the place? Should we remove all, maybe lets configure a logger and replace the prints with logging instead?

Reply:
Depends on what you're printing. They would also need to be consistent across examples, I'd expect. But yes, go ahead, from my part. Do remove the printing of the top k I saw somewhere, as it pollutes the CI check